### ISSUE TITLE
Plans for submitting LINS to the GAP package distribution?

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/LINS#50]
It would be great if this package was submitted for inclusion within the GAP package distribution.

Are there any blockers for this from your point of view?

**Comment by hulpke** [comment]
I think even without any bells and whistles, a basic "Low Index Normal Subgroups" function would be a most welcome addition -- people ask for it regularly.

**Comment by fingolfin** [comment]
Still think this would be nice to have this published in the GAP distro. If you are not sure whether you have time to maintain the package, you could always add "The GAP Team" to the list of maintainers in the `PackageInfo.g` (see e.g. [here](https://github.com/gap-packages/polymaking/blob/master/PackageInfo.g) for an example), and transfer this repository to `gap-packages`.

**Comment by FriedrichRober** [comment]
Yes, let's publish this on the GAP distro and move it to `gap-packages`. It doesn't seem likely that I will do the traversing suff due to time constraints. I am currently sick, so I will contact you on slack when I feel better and you can show me what to do.

**Comment by hulpke** [comment]
I'm very much in favor of distributing the package. Would it be possible to add a trivial function

```
LowIndexNormalSubgroups:=function(g,n)
local l;
  l:=LowIndexNormalSubgroupsSearchForAll(g,n);
  return List(List(l),Grp);
end;
```

to cater to users who just want a list of actual subgroups without the fancier features?

**Comment by FriedrichRober** [comment]
There was a name conflict with `polycyclic`. If we can fix this, then yes.

**Comment by fingolfin** [comment]
We have migrated LINS to the `gap-packages` organization, and some work was begun to prepare for a release.

**Comment by esabo** [comment]
If it counts, I use this package here https://github.com/esabo/CodingTheory/blob/master/src/tilings.jl and would like to see it in the distro when possible. Thanks :)
